docs/02.API-REFERENCE.md

```diff
@@ -5179,7 +5179,8 @@ You can get them by calling jerry_get_object_native_pointer later.
 
 *Note*: If a non-NULL free callback is specified in the native type information,
         it will be called by the garbage collector when the object is freed.
-        The type info is always overwrites the previous value, so passing
+        This callback **must not** invoke API functions.
+        The type info always overwrites the previous value, so passing
```

Comment:
Just spotted: the doc comment of `jerry_set_object_native_pointer` in jerry.c is not updated accordingly.

Reply:
Thanks, I've forgotten it.